doc/Requirements Specification/Parts/Environment.lyx

```diff
@@ -109,7 +109,7 @@ name "/E10/"
 
 \end_inset
 
- Beagle requires Java and Eclipse to run.
+ Beagle requires not less than Java 7 and Eclipse Mars to run.
```

Comment:
Beagle should run on a Java 7 runtime environment (or higher) and on an Eclipse distribution that is up to date with Eclipse Mars (4.5).